### ISSUE TITLE
Using kauth for privilege elevation

### ISSUE THREAD

**Issue by bobbobbobbobbobbobbobbobbobbob** [KDAB/hotspot#138]
kdesudo is not maintained anymore and also not part of ubuntu 18.04.

Please use kauth for privilege elevation.

**Comment by milianw** [comment]
yes, that would be good - but it's not easy to do this. Esp. for local testing I couldn't find a way. I.e. kauth helpers apparently must be installed system wide, which is bad for my dev setup.

help wanted!
